internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go

```diff
@@ -0,0 +1,141 @@
+//go:build integration || localintegration || temporary
```

Comment:
Does it make more sense for the file name to be `nlb_web_service_integration_test`?

Reply:
I was thinking this as well but wanted to be grouped with the other `lb_` tests (you know how editor lists the files alphabetically). But I'm fine with either!